dv/uvm/core_ibex/scripts/run_rtl.py

```diff
@@ -80,17 +81,22 @@ def _main() -> int:
     trr.dir_test.mkdir(exist_ok=True, parents=True)
     trr.rtl_cmds   = [format_to_cmd(cmd) for cmd in sim_cmds]
     trr.rtl_stdout = trr.dir_test / 'rtl_sim_stdstreams.log'
-    trr.export(write_yaml=True)
 
     # Write all sim_cmd output into a single logfile
     with open(trr.rtl_stdout, 'wb') as sim_fd:
 
-        for cmd in trr.rtl_cmds:
-            # Note that we don't capture the success or failure of the subprocess:
-
-            sim_fd.write(f"Running run-rtl command :\n{' '.join(cmd)}\n".encode())
-            run_one(md.verbose, cmd, redirect_stdstreams=sim_fd, timeout_s=1800)
+        try:
+            for cmd in trr.rtl_cmds:
+                # Note that we don't capture the success or failure of the subprocess:
+                sim_fd.write(f"Running run-rtl command :\n{' '.join(cmd)}\n".encode())
+                run_one(md.verbose, cmd,
+                        redirect_stdstreams=sim_fd,
+                        timeout_s=md.run_rtl_timeout_s,
```

Comment:
Is this now a different timeout to the one that UVM uses?

Reply:
Yes this was always a different timeout to the UVM one, this one wraps the spawned process for the simulation and kills it if it does not terminate on its own.

Reply:
So I think this value is currently set to 30 minutes. Is that guaranteed to be longer than any current UVM timeout?